ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -4,5 +4,6 @@
     <filter root="/apps/sling"/>	
     <filter root="/etc/acs-commons/lists/coupon-list" mode="merge"/>
     <filter root="/etc/notification/email/hclecomm" mode="merge"/>
-	  <filter root="/apps/msm/hclecomm_blueprint" mode="merge"/>
+	<filter root="/apps/msm/hclecomm_blueprint" mode="merge"/>
```

Comment:
@rakeshkumar-verma : why do we have this as part of current PR ?

Reply:
@rishabh-behl  form submit action updated in filter.